diff -urN mod_midgard-1.4.1_5/midgard.conf mod_midgard-1.4.1_5.grzegorz/midgard.conf
--- mod_midgard-1.4.1_5/midgard.conf	Sat Apr 14 16:43:32 2001
+++ mod_midgard-1.4.1_5.grzegorz/midgard.conf	Thu Jun 21 17:23:43 2001
@@ -15,13 +15,13 @@
 #
 #MidgardRootfile libexec/midgard-root.php3
 #or
-#MidgardRootfile libexec/midgard-root.php4
+MidgardRootfile lib/apache/midgard-root.php
 #depending on what you'll use.
 
 # Use the MidgardParser directive to specify the behaviour of Midgard's
 # text formating engine. 
 # Default is latin1
-#MidgardParser latin1
+MidgardParser latin2
 
 # Use MidgardDefaultRealm directive in SiteGroupped environments to change
 # default realm used by Midgard. By default it is equal to "Midgard" and
